clio-transfer-model/src/main/scala/org/broadinstitute/clio/transfer/model/package.scala

```diff
@@ -0,0 +1,5 @@
+package org.broadinstitute.clio.transfer
+
+package object model {
+  def forceString: String = "force"
```

Comment:
is there a better place for this?

Reply:
Not yet, but I'd say we should create one. We could make an `ApiConstants` object to hold magic strings like this (and we could toss `"api"` in there too). package objects are usually used for type aliases / implicits.